ksql-cli/src/main/java/io/confluent/ksql/cli/commands/Local.java

```diff
@@ -98,6 +101,14 @@
   )
   String propertiesFile;
 
+  @Option(
+      name = SCHEMA_REGISTRY_URL_OPTION_NAME,
+      description = "The url of the schema registry server to be used. It defaults to " +
+                    SCHEMA_REGISTRY_URL_OPTION_DEFAULT + ". Avro support requires a functioning "
+                    + "schema registry which the KSQL instance can connect to."
+  )
+  String schemaRegistryUrl;
+
```

Comment:
Also mention that setting the schema registry property here will overwrite the one in the properties file.